### PR TITLE
Test/cfd 258 fix scheme op UI tests

### PR DESCRIPTION
**Pull request by Aaron-Collis-IW**
# Description

-   Added tests to run through as a scheme operator in cypress

# Testing instructions

-   run cypress UI test locally

# Type of change

-   [ ] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [x] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

# Checklist:

-   [x] Able to run pr locally
-   [x] Followed acceptance criteria
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have made corresponding changes to the documentation if applicable
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,5 +37,6 @@ run-cypress-edge:
 
 run-cypress-all: run-cypress-chrome run-cypress-firefox run-cypress-edge
 
+
```
Comment (jacsamell) [inline Makefile:40]: Whitespace 😱

File: cypress_tests/cypress/support/steps.ts [hunk 2]
```diff
@@ -29,21 +29,21 @@ export const defineUserTypeAndTimeRestrictions = (): void => {
 
 export const selectFareType = (
     fareType: 'single' | 'period' | 'return' | 'flatFare' | 'multiOperator' | 'schoolService',
+    isScheme: boolean,
```
Comment (jacsamell) [inline cypress_tests/cypress/support/steps.ts:32]: FYI: if you wanted you could have defaulted this to false so you didn't have to update all the references 🙂

File: cypress_tests/package-lock.json [hunk 3]
```diff
@@ -1,9507 +1,8 @@
 {
     "name": "cypress",
     "version": "1.0.0",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```
Comment (jacsamell) [inline cypress_tests/package-lock.json:4]: Think this will cause problems updating the lockfile version? Not sure why this has been updated?

**Review by jacsamell — CHANGES_REQUESTED** [review]
Looking good, couple comments.

Also think your commit message has messed up mate

File: cypress_tests/package-lock.json (hunk 3 above)
Comment (Aaron-Collis-IW) [reply cypress_tests/package-lock.json:4]: I didn't realise I'd changed this, I'll change it back 👍

File: cypress_tests/cypress/support/steps.ts (hunk 2 above)
Comment (Aaron-Collis-IW) [reply cypress_tests/cypress/support/steps.ts:32]: Good to know, will keep in mind for next time

File: Makefile (hunk 1 above)
Comment (Aaron-Collis-IW) [reply Makefile:40]: 🙈  I'll sort it

**Review by jacsamell — APPROVED** [review]

**Pull request merged**